api/tests/test-router.js

```diff
@@ -507,7 +507,6 @@ o.spec("route", function() {
 				})
 
 				o("accepts RouteResolver without `onmatch` method as payload", function() {
-					var renderCount = 0
```

Comment:
Likewise, this is a missed assertion for `renderCount === 1`, it should not be removed, but an assertion added.